tests/utils/admincli/cmd.go

```diff
@@ -30,6 +30,25 @@ func CreatePolicy(ip, name, content string) (string, error) {
 	return ssh.InvokeCommand(ip, admincli.CreatePolicy+" --name "+name+" --content "+content)
 }
 
+// RemovePolicy removes a policy.
+func RemovePolicy(ip, policyName string) (string, error) {
+	log.Printf("Removing policy [%s] on esx [%s]\n", policyName, ip)
+	return ssh.InvokeCommand(ip, admincli.RemovePolicy+" --name "+policyName)
+}
+
+// VerifyActiveFromVsanPolicyListOutput is going to check, for the given vsan policy, whether the active
+// column returned by "vmdkops policy ls" command matches the value specified by input param "active"
+func VerifyActiveFromVsanPolicyListOutput(ip, policyName, active string) bool {
```

Comment:
This function is not test whether the vsan policy is in use. It need to test whether the active column from vsan policy output is the same as expected value.